Lecture_9/src/Person.js

```diff
@@ -0,0 +1,14 @@
+export class Person {
+  constructor (firstName = 'Jon', lastName = 'Snow') {
+    this.firstName = firstName
+    this.lastName = lastName
+  }
+
+  greeting (firstName, lastName) {
```

Comment:
Seems like these parameters are unused
```suggestion
  greeting () {
```